integration_tests/dbt_project.yml

```diff
@@ -8,33 +8,33 @@ vars:
 
   ## Use this variable to run the project with synthetic data loaded as seeds
   # otherwise set to false and set the input vars below
-  test_data_override: true
+  test_data_override: false
```

Comment:
Do we not want to leave this enabled by default? I think it's nice to have for outside contributors. Makes things easier for them to test, and we can always disable it when we want to run with different data sets.

Reply:
I think there is a problem with the synthetic data.  I am seeing duplicate claims and claim line numbers which is why I've turned it off.  I plan on opening an issue and updating Slack with more info.